README.md

```diff
@@ -23,6 +24,37 @@ This splits resources off in to a nested stack dedicated to the associated Lambd
 
 This moves resources in to a nested stack for the given resource type. If `Per Lambda` is enabled, it takes precedence over `Per Type`.
 
+### Per Lambda Group
+
+This splits resources off in to a nested stack dedicated to a set of Lambda functions and associated resources. The resources within the nested stack are declared as depending on each others to avoid the `API rate limit` error due to fast deployments. If `Per Lambda` or `Per Type` is enabled, it takes precedence over `Per Lambda Group`. In order to control the number of nested stacks, more configurations are needed:
+
+```yaml
+custom:
+  splitStacks:
+    nestedStackCount: 20 # Controls the number of created nested stacks
+    perFunction: false
+    perType: false
+    perGroupFunction: true
+    resourceParallelDeployments: 10 # Controls how much resources are deployed in parallel
```

Comment:
If we go ahead with renaming it to resourceConcurrency, I would move this example to the section below, and rename it to `Concurrency` -- just to separate the migration strategy from the concurrency controls

Reply:
Done